plutarch-docs/plutarch-docs.cabal

```diff
@@ -0,0 +1,96 @@
+cabal-version: 3.6
+name:          plutarch-docs
```

Comment:
What does this do?

Reply:
It compiles documentation in CI, so you can find out that it's not up to date anymore :)